.github/workflows/build.yaml

```diff
@@ -154,6 +154,15 @@ jobs:
 
       - run: xcodebuild -project redwood-widget-uiview-test/RedwoodWidgetUIViewTests.xcodeproj -scheme RedwoodWidgetUIViewTests -destination 'platform=iOS Simulator,name=iPhone 16,OS=18.4' test
 
+      - run: ./gradlew jsBrowserTest
```

Comment:
This runs the tests for all modules (which have browser tests enabled). You probably want to target this more locally to specific modules.